src/platform/web/ui/login/PasswordLoginView.js

```diff
@@ -18,7 +18,7 @@ import {TemplateView} from "../general/TemplateView";
 
 export class PasswordLoginView extends TemplateView {
     render(t, vm) {
-        const disabled = vm => !!vm.isBusy;
+        const disabled = vm => (!!vm.isBusy) && (!vm.isEnabled);
```

Comment:
Not a change you made but we don't really need the double bang (`!!`) here since `vm.isBusy` is already a boolean.
So this can be:
```suggestion
        const disabled = vm => vm.isBusy && !vm.isEnabled;
```